setup.py

```diff
@@ -30,7 +30,7 @@
     packages=find_packages(),
     package_data={'sphinx_gallery': ['_static/gallery.css', '_static/no_image.png',
                                      '_static/broken_example.png']},
-    scripts=['bin/copy_sphinxgallery.sh'],
+    scripts=['bin/copy_sphinxgallery.sh', 'bin/sphx_ex2nb.py'],
```

Comment:
Maybe something less cryptic like sphx_glr_example_to_notebook? Not 100% convinced so better suggestions welcome! My point is that trying to shorten names as much as possible does not make too much sense since you have tab-completion these days ...